shared/js/initApp.js

```diff
@@ -65,7 +65,9 @@ window.apps = {
       },
       onContinue: function() {
         if (lastServerResponse.videoInfo) {
-          showVideoDialog(lastServerResponse.videoInfo);
+          if (dashboard.clientState.hasSeenVideo(lastServerResponse.videoInfo.key)) {
```

Comment:
It works because I hadn't disabled the session cookie work that was persisting it. Doing that now.